install/container/ubuntu.sh

```diff
@@ -54,6 +54,7 @@ chsh -s /bin/bash tipi
 chsh -s /bin/bash tipi-large
 chsh -s /bin/bash tipi-rbe
 
+git config --system --add safe.directory "*"
```

Comment:
```suggestion
# This is required because of docker virtiofs on docker on macOS. ( VirtioFS is not handling permissions as expected. All mount permissions are owned by root regardless of chown : https://github.com/docker/for-mac/issues/6243 )
git config --system --add safe.directory "*"
```